parse-nessus-xml.py

```diff
@@ -53,7 +57,8 @@
     if vuln_report[key]["risk_factor"] != "None":
         affected_hosts = vuln_report[key]["hosts"]
         print(vuln_report[key]["full_description"])
-        if len(affected_hosts) > 4:
+#        print(vuln_report[key]["plugin_output"])
```

Comment:
This should be an option, but for now it's easier for me to uncomment this when I need verbose output.

Reply:
I'm assuming you'd have to wire up reading input from the command line to do that, which isn't difficult but would be a bit of extra effort; I've used the [`click` library](https://click.palletsprojects.com/) in the past in case you're interested!